polkadot/xcm/xcm-executor/src/traits/asset_transfer.rs

```diff
@@ -23,9 +23,8 @@ use xcm::prelude::*;
 /// Errors related to determining asset transfer support.
 #[derive(Copy, Clone, Encode, Decode, Eq, PartialEq, Debug, TypeInfo)]
 pub enum Error {
-	/// Invalid non-concrete asset.
-	NotConcrete,
```

Comment:
This error is still being used in `pallet-xcm`, in a conversion from this error type to the pallet's error type. It has to be removed from there as well.

Reply:
There's also a matching error in `pallet-xcm` called [`InvalidAssetNotConcrete`](https://github.com/paritytech/polkadot-sdk/blob/eb6f5abee64e979dba25924f71ef86d2b3ca2deb/polkadot/xcm/pallet-xcm/src/lib.rs#L539). Should be removed as well.

Reply:
I removed all the relevant code to `NotConcrete` in this commit: https://github.com/paritytech/polkadot-sdk/pull/3867/commits/d1c65ce6ca25b919eb491bb32c64f442d6260cec